src/structures/osu/Beatmap.ts

```diff
@@ -190,23 +190,23 @@ export class Beatmap
 	 */
 	public get languageString(): string
 	{
-		return titleCase(BeatmapLanguage[this.language]);
+		return titleCase(BeatmapLanguage[this.language] ?? `Unknown Language: ${this.language}`);
```

Comment:
Are we going to start using Node.JS `14.0.0^`? Because in the server we are using LTS.

Reply:
We may do that in the future, but for now there is no _need_ to.

TypeScript transpiles `foo ?? bar` down to `foo !== null || foo !== void 0 ? foo : bar`, so that is not an issue. (At least using our config)
We have been using that operator in our code base since #81.